### DISCUSSION TITLE
pulling Wyze Cam v3 stream from Docker-Wyze-Bridge in Frigate 0.12.0?

### DISCUSSION THREAD

**Discussion by Cold-Lemonade** [blakeblackshear/frigate#6318] [Ask A Question]
I have a couple Wyze Cam v3 cameras which I pull into my Home Assistant running on a RPi4 using docker-wyze-bridge. The docker-wyze-bridge makes the Wyze Cam v3 streams available on an rtsp url tied to the RPi4. Can I pull this stream into Frigate? If so, how do I do it? I tried treating it as I do my other cameras, but got an error.

Many thanks.

**Comment by NickM-27** [comment]
if it's an rtsp stream it should have no issues being pulled into frigate. Most likely the IP was wrong or the port is not open

**Comment by g13092** [comment]
My apologies if this is considered necrobumping, but @Cold-Lemonade, how'd it work out?  I've got some V2 and V3 working, with docker-wyze-bridge, but wondering if I should be using go2rtc 'also'.  Cameras are recording well, but im having some gpu lock up issues after a few days.

**Reply by Cold-Lemonade** [reply]
I did get it working, but there was a lag with the rtsp stream for Wyze cameras coming off the docker-wyze-bridge so I stopped running them through Frigate.

For what its worth, I use the Wyze cameras to monitor the inside of a vacant rental unit on the first story of my brownstone and the kitchen of the upper unit where I reside that has direct access to the outside. I cut the power to this Wyze kitchen camera when I'm home using a zwave outlet. Frigate monitors the 10 poe Dahua cameras that keep watch over the outside of my house, including my direct access backyard parking.

My brownstone is in one of the most dangerous cities in New York State. My neighbors have thanked me for putting them in. They immediately stopped the petty drug dealers from doing deals on this street in the open. And a couple of weeks ago the Frigate + Coral AI + Home Assistant stopped two guys from trying to break into my house using my fire escape when I wasn't home. It was incredible. The system worked as designed. I have active deterrence cameras and once Frigate + Coral identified the two guys (who were carrying an aluminum ladder) on my back deck, Home Assistant triggered the siren in the cameras and sent a notification with photo to my cell phone. They weren't deterred at first by the siren and one guy even looked into one of my cameras and smiled. They proceeded to put they ladder up to my fire escape. The siren kept going and one of my neighbors peeked out a window and that scared them off. Meanwhile I called the cops who arrived to find them in the neighborhood claiming to be cleaning gutters for people. Which is total bs.
